### DISCUSSION TITLE
Html Content Editor is appearing as a text area.

### DISCUSSION THREAD

**Discussion by EdwinThomas11** [serenity-is/Serenity#7141] [Help] [answered]
I possess a form that activates upon clicking a tab.

**var formPromise = Q.getFormAsync(Default.MyForm.formKey);
formPromise.then((response) => {
  this.mygrid = new Serenity.PropertyGrid(this.byId('mypropertyGrid'), {
	  idPrefix: this.idPrefix + '_',
	  items: response,
	  useCategories: true,
  });
  this.myForm = new Default.MyForm(this.idPrefix + '_');
});**

Within one of the properties, I've implemented a [HtmlContentEditor(Rows = 4)]. However, this particular property loads as a text area instead of a cke editor. I'm seeking assistance with this issue, and your help would be greatly appreciated.

**Comment by VictorTomaili** [comment, marked as answer]
Check the `MyForm.ts` file for the editor type. It likely hasn't regenerated due to another TypeScript error or some other issue.

**Reply by EdwinThomas11** [reply]
@VictorTomaili , I've noticed that the editor type specified in Default.MyForm.ts is set to HtmlContentEditor. I'm curious why it's still displaying as a text area rather than an HtmlContentEditor, especially since I haven't encountered any console errors.

Is there a debugging method available or is there a way to activate the htmlcontenteditor again from the dialog.ts file?

**Reply by EdwinThomas11** [reply]
I addressed this issue by incorporating the following code into my TypeScript file:
**this.form.myproperty = new Serenity.HtmlContentEditor(this.byId('myproperty'));**

Previously, in my form.cs file, I had declared this property as HtmlNoteContentEditor.
